docs/authentication/session-management-console.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Admin Console actions to support the various development options
+id: session-management-console
+description: ''
+slug: /session-management-console
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/15/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
```

Comment:
might be nice to have a simple explanation about what this is explaining

Reply:
Jen suggested it could either be a higher-level category or its own page. I originally made it a higher-level category, but when I removed the APIs from the dropdown, I figured I'd see how it felt as its own page, since there was nothing else in the dropdown. I can collapse the documents back into one page though.

Reply:
I think it might be nice to link to this in the first document. I think you do link to is once, but maybe in a more obvious way that here's how to configure this stuff in the admin console.